### ISSUE TITLE
After I update the windows system, the signalr client could not connect the server any more

### ISSUE THREAD

**Issue by nkfeiyang** [SignalR/SignalR#4338]
My windows version is win7 64 sp1. After I update my OS today, The client could not connect to the server. Even when I use the vs to host on my local machine, the client still could not work.
Did any one encounter the same problem?

I have use the telnet to check, the server port is good to open. And from other machine, the clients can connect to the server.

I tried to restore my OS to 3/13, the problem disappeared, but when I roll back to the current status, the problem occurred again.

**Comment by halter73** [comment]
We're not aware of any issues related to Windows updates. Did you update your client or your server?

**Comment by nkfeiyang** [comment]
> We're not aware of any issues related to Windows updates. Did you update your client or your server?

No any signalr client and server update. It is suddenly my client disconnect to the server and could not connect again. I checked my windows update and found at the same time, there are many updates related to the Office.

I tried to use another windows 7 computer, update the system, but could not reproduce the issue. Maybe my computer's unknown issue caused this problem.
